ApplicationLibCode/ProjectDataModel/Completions/RimEllipseFractureTemplate.cpp

```diff
@@ -402,12 +422,18 @@ void RimEllipseFractureTemplate::convertToUnitSystem( RiaDefines::EclipseUnitSys
         m_halfLength = RiaEclipseUnitTools::meterToFeet( m_halfLength );
         m_height     = RiaEclipseUnitTools::meterToFeet( m_height );
         m_width      = RiaEclipseUnitTools::meterToInch( m_width );
+
+        // Convert here instead of base class to avoid interfering with StimPlan template
+        // which handles units differently for this property.
+        m_wellPathDepthAtFracture = RiaEclipseUnitTools::feetToMeter( m_wellPathDepthAtFracture );
```

Comment:
Should be meterToFeet?

Reply:
Good catch! Probably cut-and-paste slip.. Fixed no.